src/goto-instrument/contracts/instrument_spec_assigns.h

```diff
@@ -451,6 +451,17 @@ class instrument_spec_assignst
     skip_function_paramst skip_function_params,
     optionalt<cfg_infot> &cfg_info_opt);
 
+  template <typename C>
+  void get_static_locals(std::insert_iterator<C> inserter) const
```

Comment:
Where is the documentation header for this method?

Reply:
Good point, I'll add doc